test/library/packages/Sort/Radix_Sort_LSB_test/lsb.chpl

```diff
@@ -0,0 +1,72 @@
+proc countsort(place:int,array:[] int) {
+
+  var length_arr = array.size;
+  var result:[1..length_arr] int;
+  var counts:[0..256] int;
+
+  var i: int;
+  var current_digit: int;
+  
+
+  for i in 1..array.size{
+    // find the postion to store a particular number in the counts array
+    current_digit = ((array[i] >> place) & 255); 
+    //increment count at that position
+    counts[current_digit] += 1;
+  }
+
+  i=1;
+  for i in 1..(counts.size-1) {
+    counts[i] += counts[i-1];
+  }
+
+  i = array.size;
+  var new_index: int;
+
+  // fill in the result array with the correct position of the numbers using the counts array.
+  while(i>0) {
+    
+    current_digit = (array[i] >> place) & 255; 
+    new_index = counts[current_digit];
+    result[new_index] = array[i];
+    counts[current_digit] -=  1;
+    i = i - 1;
```

Comment:
This one goes through the input and output data in reverse but I'd like it to go forward because I expect it to be more efficient. You might need to create two variables - counts and offsets, say. 

Also I see that gathering the counts and doing a stable shuffle are common between the MSB and LSB versions. Could we make them into subroutines that they both can call?